pullpigo.go

```diff
@@ -26,16 +27,21 @@ func main() {
 	}
 }
 
+type config struct {
```

Comment:
What about naming conventions (case)? 🤔

Reply:
Seems OK with https://golang.org/doc/effective_go.html#names
Camel case seems OK since this structure is not exported.